### DISCUSSION TITLE
Gopro Wide Gamut IDT / Davinci Wide Gamut conversion

### DISCUSSION THREAD

**Discussion by markusfinholt** [gopro/labs#481] [Q&A] [answered]
Hi, has anyone been able to create a gopro wide gamut to dwg / aces idt yet?

**Comment by andrerav** [comment]
Hi @markusfinholt,

I can warmly recommend the technical LUTs from Paul Leeming. He has a profile specifically for LOGB=400 and WIDE=1. 

- [Example video 1](https://www.youtube.com/watch?v=AAzottipCfs)
- [Example video 2](https://www.youtube.com/watch?v=AtPYA8JJVR0)

And a screengrab from a video I'm editing right now:

![image](https://github.com/gopro/labs/assets/3635018/dfded9ac-0ad2-411c-9e21-6eb7df52f30a)

**Reply by markusfinholt** [reply]
Unfortunately a LUT from Wide to rec709 will only allow a display referred workflow, not a scene referred one.

**Comment by dnewman-gpsw** [comment, marked as answer]
This user seems know the steps to compute IDT for GoPro shoot modes. https://github.com/xtremestuff/protune-transforms
